### PR TITLE
Change the default RGB light effects.

### PR DESCRIPTION
**Pull request by mj-bomb**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/melgeek/mj6xy/rev3/config.h [hunk 1]
```diff
@@ -41,7 +41,10 @@
 #endif
 
 #ifdef RGB_DI_PIN
-#    define RGBLIGHT_ANIMATIONS
+#	 define RGBLIGHT_ANIMATIONS
+#	 define RGBLIGHT_EFFECT_RAINBOW_MOOD
```
Comment (drashna) [inline keyboards/melgeek/mj6xy/rev3/config.h:45]: if `RGBLIGHT_ANIMATIONS` is defined, you don't need this defined, as well,
```suggestion
```

**Comment by mj-bomb** [comment]
@tzarc @fauxpark   Hello, my friend. Can you vet it for me

**Comment by mj-bomb** [comment]
@tzarc @fauxpark   Hello, my friend. Could you mind review for my PR?

File: keyboards/melgeek/mj6xy/rev3/config.h [hunk 2]
```diff
@@ -41,7 +41,9 @@
 #endif
 
 #ifdef RGB_DI_PIN
-#    define RGBLIGHT_ANIMATIONS
+#	 define RGBLIGHT_ANIMATIONS
+#	 define RGBLIGHT_DEFAULT_MODE (RGBLIGHT_EFFECT_RAINBOW_MOOD + 6)
```
Comment (fauxpark) [inline keyboards/melgeek/mj6xy/rev3/config.h:45]:
```suggestion
#    define RGBLIGHT_ANIMATIONS
#    define RGBLIGHT_DEFAULT_MODE (RGBLIGHT_EFFECT_RAINBOW_MOOD + 6) 
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**